rag-langchain-python/vector_query_retriever.py

```diff
@@ -45,6 +46,7 @@ def _build_eval_script(self, query, query_embedding):
     op.vec.vector(op.col('embedding')),
     op.vec.vector(vec.vector({}))
   )))
+  .select([op.col('uri'), op.col('transcript'), op.col('similarity')])
```

Comment:
I'm never clear on when `op.col` is required and when a simple string works - but try `select(['uri', 'transcript', 'similarity'])` to see if it works.

Reply:
works: .select(['uri', 'transcript', 'similarity'])